docs/tools/debugging/pro.md

```diff
@@ -0,0 +1,106 @@
+---
+order: 2
+---
+
+# Debug Like A Pro
```

Comment:
A couple of things about this page:
1. Is this page linked to from anywhere else? I can't seem to find a link to it anywhere.
2. This page seems to be written in the style of a blog post, which seems kind of out of character given the nature of the rest of the documentation in the docs site. I'd recommend perhaps paring this down a bit to make it more appropriate as technical documentation.

Reply:
I think this was initially written by Anton a long time ago (and was a blog post as well as part of tendermint docs)

Reply:
Removed. there is sufficient information in the [debugging.md](https://github.com/tendermint/tendermint/blob/0e123ede6bf484cb5e637528a8cde74040ea52de/docs/tools/debugging.md#L61) file.